apps/aecore/test/aecore_naming_test.exs

```diff
@@ -412,14 +387,13 @@ defmodule AecoreNamingTest do
     assert first_name_update.pointers == "{\"test\": 2}"
 
     transfer_to_pub = setup.a_pub_key
-    {:ok, transfer} = Account.name_transfer("test.aet", transfer_to_pub, 5)
-    Pool.add_transaction(transfer)
+    Account.name_transfer("test.aet", transfer_to_pub, 5)
     Miner.mine_sync_block_to_chain()
 
     naming_state_4 = Chain.chain_state().naming
 
     assert 1 == naming_state_4 |> PatriciaMerkleTree.all_keys() |> Enum.count()
-    hash_1 = transfer.data.payload.hash.value
+    {:ok, hash_1} = NameUtil.normalized_namehash("test.aet")
```

Comment:
The data from the original transaction should be used, so in case there is an issue there we will catch it here